charts/aws/templates/aws.yaml

```diff
@@ -8,8 +8,6 @@ spec:
   aws:
   - compliance: {{ .Values.compliance }}
 
-    region: {{ .Values.regions }}
```

Comment:
region comes from `Values.connectionDetails`. This was generating a YAML to two `region` fields